libzstd/encoder/src/lib.rs

```diff
@@ -0,0 +1,46 @@
+use zstd::stream::Encoder;
```

Comment:
could we also use the param setting "inside" `.so` file, because when upgrading `zstd`, also need to sync params in this file in the meantime.